examples/gcoap_block_server/gcoap_block.c

```diff
@@ -73,6 +75,8 @@ static ssize_t _riot_block2_handler(coap_pkt_t *pdu, uint8_t *buf, size_t len, c
     plen += coap_blockwise_put_char(&slicer, buf+plen, 'U');
     plen += coap_blockwise_put_char(&slicer, buf+plen, '.');
 
+    coap_opt_replace_etag(pdu, test_etag, 4);
```

Comment:
To have an actual test of that function it'd be great if it could be verified at some point that its value is indeed AABBCCDD.

Reply:
You mean that my screen shot of wireshark is not good enough? :laughing: 

I can extend the nanocoap unittest application tomorrow with something.